R/porcelain.R

```diff
@@ -28,5 +28,15 @@
         porcelain::porcelain_state(root = state$root),
         returning = porcelain::porcelain_returning_json("report_parameters"),
         validate = validate)
+    },
+    "POST /report/run" = function(state, validate) {
```

Comment:
Perhaps copy in this file: https://github.com/jameel-institute/daedalus.api/blob/main/.gitattributes

and you'll get nicer rendering of diffs

Reply:
ooo nice